src/core/simd.d

```diff
@@ -479,4 +479,65 @@ version ( D_SIMD )
   }
 
   private void __prefetch(const(void*) address, ubyte encoding);
+
+  /*************************************
+   * Load unaligned vector from address.
+   * This is a compiler intrinsic.
+   * Params:
+   *    p = pointer to vector
+   * Returns:
+   *    vector
+   */
+
+  V loadUnaligned(V)(const V* p)
+  {
+        pragma(inline, true);
+        static if (is(V == double2))
+            return cast(V)__simd(XMM.LODUPD, *cast(const void16*)p);
+        else static if (is(V == float4))
+            return cast(V)__simd(XMM.LODUPS, *cast(const void16*)p);
+        else static if (is(V == void16) ||
+                        is(V == byte16) ||
+                        is(V == ubyte16) ||
+                        is(V == short8) ||
+                        is(V == ushort8) ||
+                        is(V == int4) ||
+                        is(V == uint4) ||
+                        is(V == long2) ||
+                        is(V == ulong2))
+            return cast(V)__simd(XMM.LODDQU, *cast(const void16*)p);
+        else
+            static assert(0);
```

Comment:
The conditions should be moved to the template constraint. Basically `static assert(0)` in a stdlib function is an antipattern.